datalad/core/distributed/clone.py

```diff
@@ -581,21 +581,141 @@ def postclonecfg_ria(ds, props):
     # setup publication dependency, if a corresponding special remote exists
     # and was enabled (there could be RIA stores that actually only have repos)
     # make this function be a generator
-    ria_remotes = [s for s in ds.siblings('query', result_renderer='disabled')
-                   if s.get('annex-externaltype', None) == 'ora'
-    ]
-    if not ria_remotes:
-        lgr.debug("Found no RIA special remote")
-    elif len(ria_remotes) == 1:
-        yield from ds.siblings('configure',
-                               name='origin',
-                               publish_depends=ria_remotes[0]['name'],
-                               result_filter=None,
-                               result_renderer='disabled')
-    else:
-        lgr.warning("Found multiple RIA remotes. Couldn't decide which "
-                    "publishing to origin should depend on: %s",
-                    [r['name'] for r in ria_remotes])
+    ora_remotes = [s for s in ds.siblings('query', result_renderer='disabled')
+                   if s.get('annex-externaltype', None) == 'ora']
+    if not ora_remotes and any(
+            r.get('externaltype', None) == 'ora'
+            for r in ds.repo.get_special_remotes().values()):
+        # no ORA remote autoenabled, but configuration know about at least one.
+        # Let's check origin's config for datalad.ora-remote.uuid as stored by
+        # create-sibling-ria and enable try enabling that one.
+        lgr.debug("Found no autoenabled ORA special remote. Trying to look it "
+                  "up in source config ...")
+
+        # First figure whether we cloned via SSH, HTTP or local path and then
+        # get that config file the same way:
+        config_content = None
+        scheme = props['giturl'].split(':')[0]
```

Comment:
since we are all about micro-optimizations these days,
```suggestion
        scheme = props['giturl'].split(':', 1)[0]
```